core/backend/src/ChangeSummaryManager.ts

```diff
@@ -8,15 +8,15 @@
 
 import { AccessToken, assert, DbResult, GuidString, Id64String, IModelStatus, Logger } from "@itwin/core-bentley";
 import { ChangedValueState, ChangeOpCode, ChangesetRange, IModelError, IModelVersion } from "@itwin/core-common";
-import * as path from "path";
-import { BackendLoggerCategory } from "./BackendLoggerCategory";
-import { BriefcaseManager } from "./BriefcaseManager";
-import { ECDb, ECDbOpenMode } from "./ECDb";
-import { ECSqlStatement } from "./ECSqlStatement";
-import { BriefcaseDb, IModelDb, TokenArg } from "./IModelDb";
-import { IModelHost, KnownLocations } from "./IModelHost";
-import { IModelJsFs } from "./IModelJsFs";
-import { _hubAccess, _nativeDb } from "./internal/Symbols";
+import path from "path";
```

Comment:
```suggestion
import path from "node:path";
```

Reply:
in BriefcaseManager you import * as path and here you dont; i would prefer to stay consistent